scripts/bincompare.py

```diff
@@ -69,7 +69,7 @@ def do_invoke(self, argv):
             return
 
         try:
-            memory_data = read_memory(start_addr, size)
+            memory_data = gdb.selected_inferior().read_memory(start_addr, size).tobytes()
```

Comment:
```suggestion
            memory_data = gef.memory.read(start_addr, size).tobytes()
```

See https://github.com/hugsy/gef/blob/dev/docs/api/gef.md#function-gefmemorymanagerread